src/Bridge/Symfony/SymfonyAdapter.php

```diff
@@ -43,4 +50,29 @@ public function handle(ServerRequestInterface $request) : ResponseInterface
 
         return $response;
     }
+
+    /**
+     * @param $symfonyRequest
+     */
+    private function loadSessionFromCookies($symfonyRequest): void
+    {
+        if (!is_null($symfonyRequest->cookies->get(session_name()))) {
+            $this->httpKernel->getContainer()->get('session')->setId(
+                $symfonyRequest->cookies->get(session_name())
+            );
+        }
+    }
+
+    /**
+     * @param $symfonyResponse
+     */
+    private function addSessionCookieToResponse($symfonyResponse): void
+    {
+        $symfonyResponse->headers->setCookie(
```

Comment:
will this cookie contain a http-only flag (or other security measures)?

Reply:
@staabm yes, it will. i've updated the unit test to reflect this. is there anything else you feel i need to be added for security?

Reply:
SameSite Lax oe Strict ?

Reply:
the Symfony default for the `Cookie` class is `httponly=true` and `SameSite` is null. I've added `Cookie::SAMESITE_LAX` into the adapter but I'm not 100% certain we want to override the defaults too much..?

Reply:
@staabm is it all good for you here? I haven't studied cookies in enough details to be of more help. (the rest of the PR is good to merge for me)